ocs_ci/ocs/ocp.py

```diff
@@ -310,7 +310,9 @@ def patch(self, resource_name='', params=None, type='json'):
         """
         resource_name = resource_name or self.resource_name
         params = "\'" + f"{params}" + "\'"
-        command = f"patch {self.kind} {resource_name} -n {self.namespace} -p {params} --type {type}"
+        command = f"patch {self.kind} {resource_name} -n {self.namespace} -p {params}"
+        if format_type:
+            command += f"--type {format_type}"
```

Comment:
I think you need to add extra space before '--type' so the command will be 
```
"command --type"
```
instead of:
```
"command--type"
```